pkg/model/api/v1/datasource/prometheus.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2021 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package datasource
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+var defaultPrometheusAllowedEndpoints = []HTTPAllowedEndpoint{
+	{
+		Endpoint: "/api/v1/labels",
+		Method:   http.MethodPost,
+	},
+	{
+		Endpoint: "/api/v1/series",
+		Method:   http.MethodPost,
+	},
+	{
+		Endpoint: "/api/v1/metadata",
+		Method:   http.MethodGet,
+	},
+	{
+		Endpoint: "/api/v1/query",
+		Method:   http.MethodPost,
+	},
+	{
+		Endpoint: "/api/v1/query_range",
+		Method:   http.MethodPost,
+	},
+}
```

Comment:
Probably want to add the `/api/v1/label/FOO/values` endpoint too?

Reply:
yeah that's a TODO (I forgot to add it), we should be able to managed endpoint with variable inside.

Reply:
and this is not yet used by the Proxy